mne/time_frequency/psd.py

```diff
@@ -143,6 +147,8 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
         to be <= n_fft.
     proj : bool
         Apply SSP projection vectors.
+    method : 'welch' | 'multitaper'
+        The method to use in calculating the PSD.
```

Comment:
Can you explain in one line what each method does? For details, we should add references in the manual